models/stg_iterable__event_extension.sql

```diff
@@ -0,0 +1,67 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_iterable__event_extension_tmp') }}
+
+),
+
+fields as (
+
+    select
+        /*
+        The below macro is used to generate the correct SQL for package staging models. It takes a list of columns 
+        that are expected/needed (staging_columns from dbt_iterable_source/models/tmp/) and compares it with columns 
+        in the source (source_columns from dbt_iterable_source/macros/).
+        For more information refer to our dbt_fivetran_utils documentation (https://github.com/fivetran/dbt_fivetran_utils.git).
+        */
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_iterable__event_extension_tmp')),
+                staging_columns=get_event_extension_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    select
+        _fivetran_id as event_id,
+        app_already_running,
```

Comment:
some of the fields in this table are booleans right? should prefix with `_is` and stuff

Reply:
Yeah we should rename